scripts/apps/web-publisher/controllers/WebPublisherManagerController.js

```diff
@@ -0,0 +1,68 @@
+WebPublisherManagerController.$inject = ['$scope', 'publisher', 'modal'];
+export function WebPublisherManagerController($scope, publisher, modal) {
```

Comment:
Checkout [`SuggestController`](https://github.com/superdesk/superdesk-client-core/blob/master/scripts/apps/authoring/suggest/SuggestController.js) for guidance on how to document controllers and their methods.

Please run `grunt docs` inside the client repository to make sure all of this documentation you've written displays correctly.